components/Banner/examples/supplemental/MDCBannerView.m

```diff
@@ -54,124 +67,527 @@ - (instancetype)initWithCoder:(NSCoder *)aDecoder {
 }
 
 - (void)commonBannerViewInit {
-  self.backgroundColor = [UIColor whiteColor];
+  self.backgroundColor = MDCColorFromRGB(MDCBannerDefaultBackgroundColor);
```

Comment:
Tip: `self.backgroundColor = UIColor.whiteColor`

Reply:
Addressed in https://github.com/material-components/material-components-ios/pull/6830/commits/3a9c5afbbf966c8a2ef5a6e8ad5d76641534b057#diff-9f546e78faa17de3d67395c6ef3629b2R61.